Package.swift

```diff
@@ -4,20 +4,19 @@
 import PackageDescription
 
 let package = Package(
-    name: "CSDJournal",
-    
-    providers: [
-    	.apt(["libsystemd-dev"])
-    ],
-    
-    products: [
-    	.library(
-    		name: "CSDJournal",
-    		targets: ["CSDJournal"]
-    	)
-    ],
-    
-    targets: [
-    	.target(name: "CSDJournal")
-    ]
+		name: "CSDJournal",
+		pkgConfig: "systemd",
```

Comment:
Brilliant!

Reply:
(However, is the indentation right? I guess Swift doesn't have a default indentation)